nzic/index.md

```diff
@@ -4,30 +4,30 @@ layout: nzic_page
 nzic_weight: 0
 ---
 
-The NZIC contest series, run by the New Zealand Olympiad in Informatics, will be held again this year. There will be 4 regular rounds held throughout the year. The difficulty ranges from entry-level problems to more challenging problems. To practice, sign up at [https://train.nzoi.org.nz/](https://train.nzoi.org.nz/).
+The NZIC contest series, run by the New Zealand Olympiad in Informatics, will be held again in 2017. There will be 3 regular rounds held throughout the year. The difficulty ranges from entry-level problems to more challenging problems. To practice, sign up at [https://train.nzoi.org.nz/](https://train.nzoi.org.nz/).
 
 The contest is open to all students currently enrolled in a secondary school (or intermediate) in New Zealand (students from year 7-13). There will be a scoreboard for individuals, and one for schools.
 
-### Scoreboard
+### 2017
 
-The scoreboard for rounds 1 and 2 can be found [here](https://docs.google.com/spreadsheets/d/16vOMjWpXa0ZvoMRvajdfGyNYAGWaLIPjijcrSdEfE4c/pubhtml).
+For 2017, we're making a couple of changes to the competition. Firstly, we're removing the supervision requirement. This is to lower the barrier to entry and encourage more students to take part. Cheating will result in disqualification, and really doesn't help you to learn about Informatics either.
 
-### Round 1
+Secondly, we are only running three rounds of the competition. This is so that we can focus more on creating balanced contests with good test data, and because many students are very busy towards the end of the year.
 
-In 2016, the first round will be Thursday 17th March 17 to Thursday 24th March. Each school may hold the contest anytime within these dates. Official competitors will be supervised and sit it during this time. Unofficial competitors may do this contest in the following week.
+Thirdly, we're trying to tailor the problems and problem statements towards more of a range of students. In each contest there should be a problem that is approachable to a student who has just learned Python in school, as well as a problem that is difficult for the most advanced students. We hope that contest discussions can help students work towards solving more problems over the year.
 
-### What is Informatics?
-
-In the context of this competition, informatics involves problem solving with computer programs. Problems of a mathematical nature are posed, and students will write a computer program to solve the problem. The competition tests the efficiency and correctness of the algorithms the students come up with. The use of a programming language is not the goal itself, but rather a method by which a solution is expressed.
+### Dates
 
-### What now?
+The dates for the three rounds will be as follows:
```

Comment:
I think it might be better to repeat it so that it's fewer clicks away, and because it's not likely to change? I've clarified the first thing and will put everything through when you've finished reviewing.